lib/src/tile_map_parser.dart

```diff
@@ -15,12 +14,14 @@ class TileMapParser {
     map.tileWidth = int.parse(xmlElement.getAttribute('tilewidth'));
     map.tileHeight = int.parse(xmlElement.getAttribute('tileheight'));
 
-    xmlElement.children.where((node) => node is XmlNode).forEach( (XmlNode node) {
+    xmlElement.children
+        .where((node) => node is XmlNode)
+        .forEach((XmlNode node) {
       if (!(node is XmlElement)) {
```

Comment:
why not do where node is XmlElement element directly and then for each XmlElement element? note: I'm assuming XmlElement extends XmlNode (didn't check but makes sense)